### PR TITLE
Checkout COD

### PR DESCRIPTION
**Pull request by bachtung144**
## Related Tickets:
- [#40607](https://edu-redmine.sun-asterisk.vn/issues/40607)

## WHAT this PR does?
checkout COD

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by bachtung144** [comment]
ready

**Review by hoangdv99 — APPROVED** [review]

**Comment by bachtung144** [comment]
ready

**Comment by bachtung144** [comment]
ready

File: app/Http/Livewire/CartComponent.php [hunk 1]
```diff
@@ -48,6 +48,11 @@ public function checkout()
 
     public function setAmountForCheckout()
     {
+        if (!Cart::count() > 0) {
```
Comment (manhtd-1083) [inline app/Http/Livewire/CartComponent.php:51]: sao so sánh phủ định count lớn hơn 0 là sao nhỉ o_O???

**Comment by bachtung144** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**